tests/integration_tests/functional/test_drives.py

```diff
@@ -449,6 +449,10 @@ def test_block_default_cache_old_version(test_microvm_with_ssh):
     response = test_microvm.vm.patch(state='Paused')
     assert test_microvm.api_session.is_status_no_content(response.status_code)
 
+    # Disable API timeout as the APIs for snapshot related procedures
+    # take longer.
+    test_microvm.api_session.untime()
```

Comment:
The `snapshot.create()` call that follows already calls `untime()`, so this is redunant.
If we also add it to the `pause_to_snapshot` method, we can get rid of all the new `untime()` calls in the tests.

Reply:
done